### ISSUE TITLE
LUA Error on hover players... sometimes

### ISSUE THREAD

**Issue by fubaWoW** [Oppzippy/TankMD#19]
Error:

```LUA
3x bad argument #1 to '?' (Usage: local definitionInfo = C_Traits.GetDefinitionInfo(definitionID))
[string "=[C]"]: in function `GetDefinitionInfo'
[string "@TankMD/Libs/LibGroupInSpecT-1.1-95/LibGroupInSpecT-1.1.lua"]:345: in function `GetCachedTalentInfoByID'
[string "@TankMD/Libs/LibGroupInSpecT-1.1-95/LibGroupInSpecT-1.1.lua"]:599: in function `BuildInfo'
[string "@TankMD/Libs/LibGroupInSpecT-1.1-95/LibGroupInSpecT-1.1.lua"]:654: in function `eventhandler'
[string "@TankMD/Libs/LibGroupInSpecT-1.1-95/LibGroupInSpecT-1.1.lua"]:131: in function <...kMD/Libs/LibGroupInSpecT-1.1/LibGroupInSpecT-1.1.lua:129>

Locals:
(*temporary) = "bad argument #1 to '?' (Usage: local definitionInfo = C_Traits.GetDefinitionInfo(definitionID))"

```

**Comment by Oppzippy** [comment]
I was hoping LibGroupInSpecT would be updated by this point, but unfortunately not. I'll do a release later today removing LibGroupInSpecT, and then look into fixing the bug so it can be re-added in the future.

**Comment by Oppzippy** [comment]
Fixed in v3.2.1

**Comment by fubaWoW** [comment]
thanks
